### ISSUE TITLE
Adding zoom factor to initialization options

### ISSUE THREAD

**Issue by ivan-katkov** [cds-astro/aladin-lite#209]
Currently, the default zooming step is too large (for my usecase) making it difficult to fine tuning the size of the view region. However, for viewing very large fields, the current zoom factor can be more appropriate. The ability to adjust the zoom factor/step could address this.

**Comment by bmatthieu3** [comment]
@ivan-katkov - I am not sure to fully understand because you can already set the fov in degrees at intialization.
Do you mean that the zooming is too fast (i.e. zoom step is too big) when zooming with a big fov ? Can you add more precision, thank you.

**Comment by tboch** [comment]
might be linked to #202

**Comment by imbasimba** [comment]
I agree that the zoom-in/out steps are a bit excessive. I often find myself zooming in too much.

**Comment by bmatthieu3** [comment]
We will provide a new listener that will be triggered when a 'wheel' event is detected on the aladin div. This will allow users to define their own zooming. Added to that, there will be 2 different "zoom" methods you can call on the Aladin object:

* `aladin.setFoV` that set the view to a specific field of view in degrees (it was there before)
* `aladin.setZoomFactor` that directly set the screen zoom factor of the view. You can see the zoom factor like a simple scaling factor on the screen.

Default zooming will uses setZoomFactor which is more stable because it allows to zoom directly on the screen and not on the world coordinates space (lon, lat) which is very projection dependant (this is the point of #202).

This work will be done in #249

**Comment by ivan-katkov** [comment]
Sorry for th silience. I think `aladin.setZoomFactor` is exactly what I had in mind. Thank you!

Just for the record, here is the trick I used to imitate the desired behavior without using `aladin.setZoomFactor`:

```js
        ///////////////////////////////////////////////////////////////////////////////////////////
        // making zooming more smooth

        const originalZoomApply = aladin.view.zoom.apply

        aladin.view.increaseZoom = function () {
            this.zoom.apply({
                stop: this.fov / 1.2,
                duration: 200
            });
        }

        aladin.view.decreaseZoom = function () {
            this.zoom.apply({
                stop: this.fov * 1.2,
                duration: 200
            });
        }

        aladin.view.zoom.apply = function(args) {
            let newArgs
            if (this.view.wheelTriggered) {
                const newFov = this.view.delta > 0 ? this.view.fov * 1.2 : this.view.fov / 1.2
                newArgs = { ...args, stop: newFov, duration: 100 }
            } else {
                newArgs = { ...args }
            }
    
            return originalZoomApply.apply(this, [newArgs])
        }
```

**Comment by bmatthieu3** [comment]
Thank you @ivan-katkov . I just merged the #249 and the features will be ready for the next 3.7.0-beta
You will be able to define your own own zooming method like so:

```js
var aladin = A.aladin('#aladin-lite-div', {target: "m51"});

// Define your own zooming by listening to wheelTriggered
aladin.on('wheelTriggered', (e) => {
    // Your zooming algo.
    // Here is a very simple example that does not take into account trackpad
    const delta = e.deltaY || e.detail || (-e.wheelDelta);
    const newZoomFactor = delta > 0 ? aladin.getZoomFactor() * 1.2 : aladin.getZoomFactor() / 1.2
    aladin.setZoomFactor(newZoomFactor)
});
```

Here is a link to the API doc listing the events you can listen to: https://cds-astro.github.io/aladin-lite/global.html#EventListener

**Comment by ivan-katkov** [comment]
Cool! Thank you!
